### PR TITLE
Update fundamentals.html

### PR DESCRIPTION
**Pull request by Zearin**
- Mark up table header cells as such
- Fix erroneous `class` attribute value for `code` element (near the beginning)
- A couple of minor punctuation copy-edits

### CODE REVIEW

File: src/fundamentals.html [hunk 1]
```diff
@@ -409,7 +409,7 @@ <h5 id="fund_units">Length Valued Attributes</h5>
 
     <p>In addition, the attributes on <a href="#presm_mpadded">mpadded</a>
      allow three <dfn>pseudo-units</dfn>, <code>height</code>,
-      <code>depth</code> and <code>width</code> (taking the place of one of the usual CSS units)
+      <code>depth</code>, and <code>width</code> (taking the place of one of the usual CSS units)
```
Comment (dginev) [inline src/fundamentals.html:412]: Do we have an official position on the use of the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) in the specification?

Should we?

Comment (davidcarlisle) [reply src/fundamentals.html:412]: @dginev I don't think we do, or are necessarily consistent. It does appear elsewhere in the spec, eg

>  Identifiers can include variables, function names, and symbolic constants

I don't like it much, despite living in Oxford, but I can live with it either way.

**Review by davidcarlisle — APPROVED** [review]

**Comment by davidcarlisle** [comment]
@dginev  I think this got lost in internal discussions. OK to merge?

**Comment by dginev** [comment]
> @dginev I think this got lost in internal discussions. OK to merge?

Certainly OK to merge. I was mostly fishing for clarification - which I got - thanks!

**Comment by davidcarlisle** [comment]
@Zearin   thanks

**Pull request merged**